code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -74,6 +74,32 @@
 	desc = "Nieaktywny dron naprawczy z wbudowanym holoprojektorem czapek."
 	drone_type = /mob/living/simple_animal/drone/snowflake
 
+/obj/item/drone_shell/attack_ghost(mob/user)
```

Comment:
co to za nowy proc?

Reply:
Nie jest nowy tylko kopiuj-wklej naprawionego archeokodu z drones_as_items.dm, z jakiegoś powodu warianty dronów go nie dziedziczyły, więc no. Nie jest to najbardziej subtelne rozwiązanie, ale nic lepszego na moim poziomie nie wymyśliłem.

Ale działa!

Reply:
coś mi tu nie pasi, są 2 proce o dokładnie tym samym typepath (nazwie) `/obj/item/drone_shell/attack_ghost`
napewno nie działałoby bez tego duplikatu w extra_drone_types.dm ?

Reply:
FUCK, commita mi jednego zeżarło, powinno być `/obj/item/drone_shell/snowflake/attack_ghost`.

Ale poza tym tak, musiałem dwa razy, bo jak nie było tej kopiujwklejki to drony miały działający limit, ale wszystkie podtypy (łącznie ze snowflake których używamy jako roundstart) już nie.

Reply:
czyli bez osobnego proca snowflake itd nie maja tego MINIMUM_AGE tak?

Reply:
Nie wiem czemu, ale tak. Innym niż snowflake nie dawałem, bo nie są raczej problemem. Dostanie się na stację jako dron z KS13 wymaga więcej wysiłku niż przeciętny griefer posiada attention spanu, plus musi wykulać KS13 w ruinach.

A istotą istnienia syndronów jest grief więc cokolwiek.